cluster/charts/rook-ceph/templates/resources.yaml

```diff
@@ -373,6 +373,21 @@ spec:
     singular: cephblockpool
   scope: Namespaced
   version: v1
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          properties:
+            failureDomain:
+                type: string
+            replicated:
+              properties:
+                size:
+                  type: integer
+                  minimum: 1
+                  maximum: 9
+                targetSizeRatio:
+                  type: string
```

Comment:
Just wondering, there is no float type in OpenAPI validation?

Reply:
@leseb Are we just passing the string value through to the ceph command and allowing ceph to validate that it's a valid value?